src/manager/node.c

```diff
@@ -1511,6 +1513,50 @@ static int node_method_reload_unit(sd_bus_message *m, void *userdata, UNUSED sd_
         return node_run_unit_lifecycle_method(m, (Node *) userdata, "reload", "ReloadUnit");
 }
 
+/*************************************************************************
+ ********** org.containers.hirte.Node.SetLogLevel *******************
+ ************************************************************************/
+
+static int node_method_set_log_level(sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bus_error *ret_error) {
+        const char *level = NULL;
+        Node *node = (Node *) userdata;
+        sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_sd_bus_message_ sd_bus_message *sub_m = NULL;
+
+        int r = sd_bus_message_read(m, "s", &level);
+        if (r < 0) {
+                hirte_log_errorf("Failed to set log level call: %s", error.message);
+                r = sd_bus_reply_method_return(m, "");
+                if (r < 0) {
+                        hirte_log_errorf("Failed to send reply to proxy request: %s", strerror(-r));
+                }
+        }
```

Comment:
```suggestion
        if (r < 0) {
                return sd_bus_reply_method_errorf(m, SD_BUS_ERROR_INVALID_ARGS, "Invalid arguments");
        }
```
Just as in the manager, just return with an `sd_bus_reply_method_errorf` stating invalid arguments.